src/ec/packages/ec-component-tag/tag.story.js

```diff
@@ -35,10 +35,11 @@ const prepareTag = (data, link, aria) => {
     );
   }
   if (data.default_icon_path) {
-    data.default_icon_path = select(
+    data.default_icon_path = optionsKnob(
       'default_icon_path',
-      [defaultSprite],
+      { current: defaultSprite, 'no path': '' },
```

Comment:
This one doesn't look quite right on toggling
![disappearing-knob](https://user-images.githubusercontent.com/1923476/86328031-7847f080-bc44-11ea-9691-27249ba0aefb.gif)